app/src/components/Blog.jsx

```diff
@@ -1,17 +1,36 @@
 import React, { Component } from 'react';
 import Footer from './Shared/Footer';
 import { Link } from 'react-router';
-import listposts from '../../styles/components/c-list-posts.scss';
+import listPosts from '../../styles/components/c-list-posts.scss';
 import CoverPrimary from './Shared/CoverPrimary';
 import PaginationBlog from './Blog/Pagination';
 import boxtriangle from '../../assets/icons/box_triangle.svg';
 
 class Blog extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      totalPageCount: 0
+    };
+
+    this.boundOnPageChange = this.onPageChange.bind(this);
```

Comment:
that's what we've been doing, yes, but to me that's kind of a smell. We are basically having two things with the same name, the bound and the unbound function. Thoughts?